### PR TITLE
Pay Button generator

### PR DESCRIPTION
**Pull request by rockstardev**

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Quite simple I like it, testing

File: BTCPayServer/BTCPayServer.csproj [hunk 1]
```diff
@@ -117,6 +117,13 @@
 
 	<ItemGroup>
 	  <Content Update="Views\Server\SSHService.cshtml">
+	  <Content Update="Views\Apps\PayButton.cshtml">
```
[the remark this answers is not in the document]
Comment (rockstardev) [reply BTCPayServer/BTCPayServer.csproj]: Forgot to push fix - try now

**Comment by NicolasDorier** [comment]
I tried it, so cool!

Thoughts: 

* Do not try to piggy bag the PoS app, just create a new AppType with its own settings.
* Allow the user to enter the currency in a textbox instead of dropdown

**Comment by NicolasDorier** [comment]
I know what you think: `But this app has no data to save so why need an app`?

Well, later we would be able to implement cool rule `If App X receives payment, then do Y`.

**Comment by NicolasDorier** [comment]
nonono remember, we talked about it. Just make it a new menu in store settings

**Comment by rockstardev** [comment]
@NicolasDorier 

mmmm no no no

Will refactor now

**Comment by NicolasDorier** [comment]
image

**Comment by NicolasDorier** [comment]
I fixed some stuff, like properly escaping attributes, and making the preview an actual button. Also a bug, Server IPN was checking for an email.

**Pull request merged**